core/neat/mixins/_grid-media.scss

```diff
@@ -46,22 +52,32 @@
 ///     margin-left: 20px;
 ///   }
 ///
-///   @media only screen and (min-width: 1000px) {
+///   @media only screen and (max-width: 999px) {
 ///     .element {
 ///       width: calc(50% - 75px);
 ///       float: left;
 ///       margin-left: 50px;
 ///     }
 ///   }
+///
+///   @media only screen and (min-width: 1000px) {
+///     .element {
+///       width: calc(50% - 75px);
+///       float: left;
+///       margin-left: 70px;
+///     }
+///   }
 
-@mixin grid-media($grid) {
-  $_media: _retrieve-neat-setting($grid, media);
-  $_query: _neat-parse-media($_media);
+@mixin grid-media($grids...) {
```

Comment:
http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variable_arguments

the '...' syntax is used to specify an unknown number of variable arguments.